scripts/install_deps.sh

```diff
@@ -4,3 +4,4 @@ set -e
 gem install xcpretty
 brew update && brew bundle
 carthage checkout
+xcrun simctl list
```

Comment:
No, not needed at all. I've added it in for debugging because I couldn't find a decent list of which simulators are available on the CI box. I think I wasted about 10 builds just to figure out how to name the sim on which to run tests. 🤦‍♂